engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-file-repositories.html

```diff
@@ -0,0 +1,118 @@
+          Add
+        </a>
+      </li>
+    </ul>
+  </nav>
+
+  <div class="details-header row">
+    <div class="col-sm-8">
+      <div class="input-group">
+
+        <span class="input-group-addon">
+          <select ng-model="product" ng-options="product.name for (id, product) in products">
+          </select>
+        </span>
+
+        <input type="text"
+               class="form-control filter-input"
+               placeholder="{{ 'Filter' | translate }}"
+               ng-model="repositorySearch"/>
+
+      </div>
+    </div>
+
+    <div class="col-sm-4">
+      <button class="btn btn-primary fr"
+              ng-disabled="repositoriesTable.numSelected === 0"
+              ng-show="isState('content-views.details.repositories.file.list') && permitted('edit_content_views', contentView)"
+              ng-click="removeRepositories(contentView)">
+        <i class="fa fa-trash-o"></i>
+        <span translate>Remove Repositories</span>
+      </button>
+      <button class="btn btn-primary fr"
+              ng-disabled="repositoriesTable.numSelected === 0"
+              ng-show="isState('content-views.details.repositories.file.available') && permitted('edit_content_views', contentView)"
+              ng-click="addRepositories(contentView)">
+        <i class="fa fa-plus"></i>
+        <span translate>Add Repositories</span>
+      </button>
+    </div>
+  </div>
+
+  <div bst-table="repositoriesTable" class="nutupane">
+
+    <div ng-show="repositoriesTable.rows.length === 0 && !repositoriesTable.working">
+      <p bst-alert="info" ng-show="isState('content-views.details.repositories.file.list')">
+        <span translate>
+          There are currently no repositories associated with this Content View, you can add some by clicking on the "Add" tab above.
+        </span>
+      </p>
+      <p bst-alert="info" ng-show="isState('content-views.details.repositories.file.available')">
+        <span translate>There are currently no repositories to add to this Content View,</span>
+        <a ui-sref="products.index" translate>add some repositories.</a>
+      </p>
+    </div>
+
+    <div class="loading-mask loading-mask-panel" ng-show="repositoriesTable.working">
+      <i class="fa fa-spinner fa-spin"></i>
+      {{ "Loading..." | translate }}
+    </div>
+
+    <table ng-show="repositoriesTable.rows.length > 0"  class="table table-bordered table-striped">
+      <thead>
+        <tr bst-table-head row-select>
+          <th bst-table-column translate>Name</th>
+          <th bst-table-column translate>Product</th>
+          <th bst-table-column translate>Last Sync</th>
+          <th bst-table-column translate>Sync State</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr bst-table-row
+            row-select="repository"
+            ng-repeat="repository in repositoriesTable.rows | filter:repositorySearch | filter:repositoryFilter | as:'filteredItems'">
```

Comment:
You update to fix this didn't work:
This was walden's updates:
 +            ng-repeat="repository in repositoriesTable.rows | filter:repositorySearch | filter:repositoryFilter as filteredItems"> 

and this was yours:

 +            ng-repeat="repository in repositoriesTable.rows | filter:repositorySearch | filter:repositoryFilter | as filteredItems">